modules/docs/arrow-docs/docs/_data/menu.yml

```diff
@@ -234,6 +234,9 @@ options:
       - title: Alternative
         url: /docs/typeclasses/alternative/
 
+      - title: Hash
```

Comment:
I'd put it below `Eq` instead, to simplify that's simple, it inherits from it, and this may be somewhat common anyway :D